_data/guides.yml

```diff
@@ -60,6 +60,8 @@ toc:
     path: /docs/user-guide/working-with-resources/
   - title: Garbage collection
     path: /docs/user-guide/garbage-collector/
+  - title: Using NetworkPolicy
```

Comment:
I think there should be Calico in the title? I'd be really great to have a general intro page talking about policy API and how it works from user's point of view, that page should also link to design documents and whatever resources we have on the (SIG Slack channel, the mailing list etc), and implementation guides including this one.